### ISSUE TITLE
Could not pass event WorldInitEvent to PlotSquared v3.2.18

### ISSUE THREAD

**Issue by Aberzhulan** [IntellectualSites/PlotSquared#753]
I am using version 3.2.18 and am getting the "Could not pass event WorldInitEvent" error. I tried downgrading version a couple times but it kept happening. Version 3.2.0 does not seem to have that issue.

Here is a copy of the startup and errors from one of the only logs I can open (the rest are nearly 1gb from plot deletion spam upon stopping the server.)

Startup log & errors: http://pastebin.com/EKE502uX
Also, here is the plot debugpaste. http://hastebin.com/yufugotuga 

I forgot to run a debugpaste when I had v3.2.18, but everything should be the same except the PlotSquared version # and the latest.log, which would be in the first link. 

Thanks.

**Comment by boy0001** [comment]
Nothing has changed regarding configuration loading.

The error seems to indicate that the wall filling was set to "[1:0". The settings.yml you provided is not the same as the one used during that startup log.

**Comment by Aberzhulan** [comment]
I tried again just now, and everything went smoothly...except for the fact that my plot worlds are no longer plot worlds. I remembered to grab a debugpaste this time: http://hastebin.com/evoveroxed

My actual settings.yml is: http://pastebin.com/6r3AY2Lf

This is a settings.yml while the server is running, directly after I got that debug paste. They are exactly the same.

I don't see (and neither does find) a "[1:0" in my settings.
